ast/Makefile

```diff
@@ -0,0 +1,26 @@
+# List any generated files here
+TARGETS = fbast/*.go
```

Comment:
@jsternberg Just did some work to make the Makefile nonrecusive, I am not sure if that has been merge yet but a heads up that this will need updating at some point soon.

Reply:
It should be decently easy to integrate this with the nonrecursive Makefile. If the other PR gets merged first, I'll help Chris to update this. If this gets merged first, I'll rebase my PR.